android/java/org/chromium/chrome/browser/BraveActivity.java

```diff
@@ -32,5 +33,10 @@ public void onStartWithNative() {
         PrefServiceBridge.getInstance().setBoolean(Pref.NTP_ARTICLES_LIST_VISIBLE, false);
     }
 
+    public boolean isNoRestoreState() {
+        String PREF_CLOSE_TABS_ON_EXIT = "close_tabs_on_exit";
```

Comment:
I would declare it on top of the class as `static final`

Reply:
Addressed in https://github.com/brave/brave-core/pull/3428/commits/b4d101eb6aa4a7e0a967e68d410f7c1ecd6550a3, thanks!